vite.config.mts

```diff
@@ -28,7 +28,7 @@ export default defineConfig({
       exclude: [
         // Default
         'coverage/**',
-        'dist/**',
+        '**/dist/**',
```

Comment:
These are the the default patterns. We can replace them with [`coverageConfigDefaults.exclude`](https://vitest.dev/config/#coverage-exclude) from `vitest/config` and then add `**/dist/**` as a custom rule.